docs/tutorials/calibration/minimal_working_example.jl

```diff
@@ -0,0 +1,112 @@
+# # Calibration: "perfect model" experiment.
+
+# In this tutorial, we demonstrate that we can recover the parameters of a previous model run via our calibration framework,
+# by calibrating a single site (Ozark) latent heat flux (LHF).
+
+# First, we run a simulation of Ozark with known values of parameters g1 and g0, which control the stomatal behavior of leaves
+# and therefore the canopy transpiration, and retrieve latent heat flux (transpiration expressed in energy).
+
+# Then, we use EnsembleKalmanProcesses.jl to perform a calibration by giving a prior distribution of the parameters g1 and g0, different from
+# the known parameters used in the first "perfect model" run. Our goal is that, after calibraton on the perfect model latent heat flux, the parameters
+# will be close to the original ones, therefore demonstrating that we can recover the parameters of the perfect model.
+
+# To perform this calibration we will need:
+# 1. A function returning our model LHF output given the parameters we want to calibrate.
+# 2. The "truth" target data, to calibrate on.
+# 3. The prior distribution of these parameters.
+
+# ## Import required packages:
+import ClimaLandSimulations.Fluxnet as CLS # to run the model
+import EnsembleKalmanProcesses as EKP # to perform the calibration
+import Logging
+Logging.disable_logging(Logging.Warn); # hide julia warnings
+
+# ## Write a function returning our model LHF output given the parameters to calibrate
+function Ozark_LatentHeatFlux(params) # params is a 2 element Array
+    g1, g0 = params
+    sv = CLS.run_fluxnet(
+        "US-MOz";
+        params = CLS.ozark_default_params(;
+            conductance = CLS.conductance_ozark(; g1 = g1, g0 = g0),
+        ),
+    )[1]
+    inputs = CLS.make_inputs_df("US-MOz")[1]
+    simulation_output = CLS.make_output_df("US-MOz", sv, inputs)
+    LHF_soil =
+        [parent(sv.saveval[k].soil.turbulent_fluxes.lhf)[1] for k in 1:1441]
+    LHF_canopy = [parent(sv.saveval[k].canopy.energy.lhf)[1] for k in 1:1441]
+    LHF = LHF_soil + LHF_canopy
+    return LHF
+end;
+
+# ## "Truth" target data to calibrate on
+# Our default for Ozark, g1 in sqrt(Pa). This is equal to 4.46 sqrt(kPa).
+LHF_target = Ozark_LatentHeatFlux([141.0, 0.0001]);
+
+# ## Parameters prior
+# mean of 221 sqrt(Pa) = 7 sqrt(kPa), std of 100 (3 kPa).
```

Comment:
"We choose the prior for each parameter to be a Gaussian distribution, with preset mean and standard devation. For g1, we choose... For g0, we choose..."